### DISCUSSION TITLE
A Contributor Or A Project Admin?

### DISCUSSION THREAD

**Discussion by hritika2409** [sanjay-kv/Open-source-Practice#934] [Q&A] [answered]
What is better being a contributor or a project admin ,as currently i m selected as a contributor but now as the form of projects opened again, i do have 2 projects which are actually good for this.So what should i choose?

**Comment by sanjay-kv** [comment, marked as answer]
I suggest being contributor , if the project is big and you well phased on opensource then be a PA, but PA comes with lot of responsibility

**Reply by hritika2409** [reply]
Ok thank you

**Comment by lovelymahor** [comment]
If you are more experienced and confident in your skills, being a project admin could be a good fit. If you're still building your skills, contributing might be better.

**Comment by Yaser-123** [comment]
It depends on what you're looking to gain:

Being a contributor helps you improve your coding skills, learn teamwork, and gain experience working on established codebases.
Being a project admin allows you to lead, manage contributors, and make impactful decisions, but requires more responsibility.
